runtime/oti/jclprots.h

```diff
@@ -39,58 +39,62 @@ void JNICALL Java_com_ibm_oti_vm_VM_releasePointerToNativeResourceImpl(JNIEnv *e
 jlong JNICALL Java_com_ibm_oti_vm_VM_getPointerToNativeResourceJarImpl(JNIEnv *env, jobject recv, jstring jarPath, jstring resName, jintArray size);
 
 /* BBjclNativesCommonShared*/
-jint JNICALL 
+jboolean JNICALL
+Java_com_ibm_oti_shared_Shared_isNonBootSharingEnabledImpl(JNIEnv* env, jclass clazz);
+jboolean JNICALL
+Java_com_ibm_oti_shared_SharedAbstractHelper_getIsVerboseImpl (JNIEnv* env, jobject thisObj);
+jint JNICALL
 Java_com_ibm_oti_shared_SharedClassAbstractHelper_initializeShareableClassloaderImpl (JNIEnv* env, jobject thisObj, jobject classloader);
-jobject JNICALL
-Java_com_ibm_oti_shared_SharedDataHelperImpl_storeSharedDataImpl (JNIEnv* env, jobject thisObj, jobject loaderObj, jint helperID, jstring tokenObj, jobject byteBufferInput);
-jlong JNICALL 
+jstring JNICALL
+Java_com_ibm_oti_shared_SharedClassStatistics_cacheDirImpl(JNIEnv* env, jobject thisObj);
+jlong JNICALL
 Java_com_ibm_oti_shared_SharedClassStatistics_freeSpaceBytesImpl (JNIEnv* env, jobject thisObj);
```

Comment:
While you're tidying up, can you please remove the spaces before `(` (here, lines 45, 57, 65, ...).